server/src/app.ts

```diff
@@ -19,7 +20,7 @@ export class App {
     });
   }
 
-  private initializeControllers(controllers: Array<CalendarController>) {
+  private initializeControllers(controllers: Array<Controller>) {
     controllers.forEach((controller) => {
       this.app.use("/", controller.router);
```

Comment:
shouldn't this be this.app.use(controller. path, controller.router)?

Reply:
I think the controller.router, adds the ```controller.path``` to the ```controller.router```.  So if we use ```his.app.use(controller. path, controller.router)``` it will add the new endpoint at ```/controller.path/controller.path``` instead

Reply:
Probably when we have more than one endpoint, we can have a main path and secondary paths (app handling the main path and controller.router handling the secondary paths)

Reply:
Ok. Makes sense